apps/web/src/views/universalFarms/components/PositionItem.tsx

```diff
@@ -102,24 +132,21 @@ const PriceRange = memo(({ currencyBase, currencyQuote, priceLower, priceUpper,
   ) : null
 })
 
-export const PositionV2Item = memo(({ data, pool }: { data: V2LPDetail; pool?: PoolInfo }) => {
+export const PositionV2Item = memo(({ data }: { data: V2LPDetail; pool?: PoolInfo }) => {
```

Comment:
if we don't need poolInfo, let's remove it, or it will cause rerender, as there many fields in poolInfo will update

Reply:
noop, we need it, i'll optimize the pull pool data and then pass to it, can just left now